KBaseSets.spec

```diff
@@ -10,11 +10,35 @@ module KBaseSets {
     */
     typedef string ws_obj_id;
 
+    /*
+        A representation for any Workspace object that should be associated with a Set item.
+    */
     typedef structure {
         string name;
         ws_obj_id ref;
     } DataAttachment;
 
+
+    /****************************** */
+    /* DIFFERENTIALEXPRESSIONMATRIX SET */
+
+    /*
+        The workspace id for a FeatureSet data object.
+        @id ws KBaseFeatureValues.DifferentialExpressionMatrix
+    */
+    typedef string ws_diffexpmatrix_id;
+
+    typedef structure {
+        ws_diffexpmatrix_id ref;
+        string label;
+    } DifferentialExpressionMatrixSetItem;
+
+    typedef structure {
```

Comment:
I think we've been registering types from this file? If so, we should just add the WS metadata annotations to all the set types in this file and reregister. I see you have them in the API spec, but that won't be seen by the WS.